3-0-java-core/src/main/java/com/bobocode/file_stats/FileStats.java

```diff
@@ -0,0 +1,46 @@
+package com.bobocode.file_stats;
+
+/**
+ * {@link FileStats} provides an API that allow to get character statistic based on text file. All whitespace characters
+ * are ignored.
+ */
+public class FileStats {
+    /**
+     * Creates a new immutable {@link FileStats} objects using data from text file received as a parameter.
+     *
+     * @param fileName input text file name
+     * @return new FileStats object created from text file
+     */
+    public static FileStats from(String fileName) {
+        throw new UnsupportedOperationException("It's your job to make it work!"); //todo
```

Comment:
@maxstasiuk92 since you already have other changes to apply (test order), could you please also replace this exception with `ExecriseNotCompletedException()` (see `CrazyLambdas` for the reference)
Thanks!

Reply:
@tboychuk ok, done